Sources_v3/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -657,13 +657,15 @@ public extension _ChatChannelController {
     ///   - text: Text of the message.
     ///   - extraData: Additional extra data of the message object.
     ///   - attachments: An array of the attachments for the message.
+    ///   - quotedMessageId: An id of the message new message quotes. (inline reply)
     ///   - completion: Called when saving the message to the local DB finishes.
     ///
     func createNewMessage(
         text: String,
 //        command: String? = nil,
 //        arguments: String? = nil,
         attachments: [_ChatMessageAttachment<ExtraData>.Seed] = [],
+        quotedMessageId: MessageId? = nil,
```

Comment:
The replies should be created only via message controller of the parent message.